server/src/tests/integration/withdrawal.rs

```diff
@@ -0,0 +1,212 @@
+//! Code related to OP-stack withdrawal flow.
+//! (Separated into its own module because there is a lot of it.)
+
+use super::*;
+
+#[allow(clippy::too_many_arguments)]
+mod op_oracle {
+    alloy::sol!(
+        #[sol(rpc)]
+        L2OutputOracle,
+        "src/tests/res/L2OutputOracle.json"
```

Comment:
These contract ABI files are under `server/src/tests/optimism/packages/contracts-bedrock/snapshots/abi`.
Should we copy into `res` or refer to them?

Reply:
I think the file I have included in `res` has more information than what is in `snapshots/abi`. For example the `WithdrawalTransaction` struct definition.